Formula/pyqt5.rb

```diff
@@ -15,8 +16,8 @@ class Pyqt5 < Formula
 
   deprecated_option "enable-debug" => "with-debug"
 
-  depends_on :python3 => :recommended
-  depends_on :python => :optional
+  depends_on :python => :recommended
+  depends_on :python3 => :optional
```

Comment:
Now the bottle is always asking for Homebrew's python3, right? This is not too great, either?

I could do it as in https://github.com/Homebrew/homebrew-core/blob/master/Formula/pyqt.rb:
```
option "without-python", "Build without python 2 support"
depends_on :python3 => :optional
```
but is the loop `Language::Python.each_python(build) do |python, version|` then still taking care of python 2?